aspnetcore/security/authentication/identity-api-authorization.md

```diff
@@ -21,6 +21,7 @@ This article shows how to use Identity to secure a Web API backend for SPAs such
 
 The steps shown in this article add authentication and authorization to an ASP.NET Core Web API app that:
 
+* Isn't already configured for authentication.
 * Targets `net8.0` or later.
 * Includes OpenAPI support.
```

Comment:
Not sure we want to make this restriction. Is OpenAPI required, or can we do a callout to the differences if they choose to work on an API _without_ OpenAPI support?